docs/solution-guidance/modern-experience-site-classification.md

```diff
@@ -7,6 +7,9 @@ localization_priority: Priority
 
 # SharePoint "modern" sites classification
 
+> [!NOTE]
+> You can now use sensitivity labels (https://docs.microsoft.com/en-us/microsoft-365/compliance/sensitivity-labels-teams-groups-sites?view=o365-worldwide) instead of sites classification to help protect your SharePoint sites.

```

Comment:
Please remove the locale from the URL (MSDocs req). This is handled by the site redirecting to the user's browser defined locale.

Reply:
Completed